homeassistant/components/sutro/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "sutro",
+  "name": "Sutro",
+  "documentation": "https://github.com/ydogandjiev/hass-sutro",
+  "issue_tracker": "https://github.com/ydogandjiev/hass-sutro/issues",
+  "dependencies": [],
+  "config_flow": true,
+  "codeowners": ["@ydogandjiev"],
+  "requirements": [],
```

Comment:
Missing requirement :)

Reply:
I am not quite sure what to put in here. As far as I can tell, most other components list any custom libraries they might be using to interact with the devices; however, that doesn't exist for Sutro. This component uses aiohttp to make calls directly to Sutro's web API. Should I just remove this entry completely?

Reply:
See this review comment: <https://github.com/home-assistant/core/pull/75585#discussion_r928309448>